x-pack/plugins/osquery/cypress/e2e/all/alerts.cy.ts

```diff
@@ -76,14 +77,12 @@ describe('Alert Event Details', () => {
     cy.getBySel(RESPONSE_ACTIONS_ITEM_0).within(() => {
       cy.get(LIVE_QUERY_EDITOR);
     });
-    cy.contains('Save changes').click();
+    cy.contains('Save changes').wait(500).click();
```

Comment:
can solve it in a different way?

Reply:
Turns out the line was obsolete - we don't have to submit to trigger validation. Removed the line.